### PR TITLE
Links with React router

### PR DESCRIPTION
**Pull request by kiowd**

### CODE REVIEW

File: frontend/src/Components/AskQuestionPage.js [hunk 1]
```diff
@@ -2,12 +2,11 @@ import React from "react";
 import "bootstrap/dist/css/bootstrap.css";
 import Header from './Header';
 import QuestionForm from './QuestionForm'; 
-
+import { Link } from 'react-router-dom';
```
Comment (delamorinierejh) [inline frontend/src/Components/AskQuestionPage.js:5]: Doesn't look like you're using this in this file. It's best practice not to import things unless you're using them

**Review by delamorinierejh — COMMENTED** [review]
Great stuff - one comment. Also it looks like this is similar (or a small extension on) this PR: https://github.com/ellietms/QueueAndA/pull/7 - 

It might be a good idea to discuss which one of these PRs you want to keep open (and evenutally merge) and then close the other one. 👍 

@kiowd @Hamidreza-Fakharzadeh

File: frontend/src/Components/AskQuestionPage.js (hunk 1 above)
Comment (Hamidreza-Fakharzadeh) [reply frontend/src/Components/AskQuestionPage.js:5]: Thank you.I deleted the line also checked all components.

**Comment by Hamidreza-Fakharzadeh** [comment]
changed some tag attribute name to correct one also deleted unused import codes

**Pull request merged**